corehq/apps/data_cleaning/models.py

```diff
@@ -6,13 +6,17 @@
 from django.db import models, transaction
 from django.utils.translation import gettext_lazy, gettext as _
 
+from dimagi.utils.chunked import chunked
+
 from corehq.apps.case_search.const import METADATA_IN_REPORTS
 from corehq.apps.data_cleaning.exceptions import (
     UnsupportedActionException,
     UnsupportedFilterValueException,
 )
 from corehq.apps.es import CaseSearchES
 
+BULK_OPERATION_CHUNK_SIZE = 1000
```

Comment:
I'm curious where this number came from. Is it a stab in the dark, or based on some similar operation?

Reply:
I peeked around where `chunked` was being used and started with a low number that seemed reasonable. I think we can opt to increase it after testing if we find performance is ok with bigger chunks

https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/domain/deletion.py#L231-L242